modules/programs/vscode.nix

```diff
@@ -97,14 +97,14 @@ in {
         default = { };
         example = literalExpression ''
           {
-            "version": "2.0.0",
-            "tasks": [
+            "version" = "2.0.0";
+            "tasks" = [
               {
-                "type": "shell",
-                "label": "Hello task",
-                "command": "hello",
+                "type" = "shell";
+                "label" = "Hello task";
+                "command" = "hello";
               }
-            ]
+            ];
```

Comment:
What do you think of using the following?
```nix
{
  version = "2.0.0";
  tasks = [
    {
      type = "shell";
      label = "Hello task";
      command = "hello";
    }
  ];
}
```

Reply:
No problem, but I followed the style in settings.json above.
The tasks.json does not have a flattened key like "files.autosave", so there is less confusion without the quotes.